package.json

```diff
@@ -0,0 +1,59 @@
+{
```

Comment:
You don't ahve README file, add it. I know you don't have time to write everything. But maybe you could just add short description about what this lib solves and add link to somethnig like this:
`https://auth0.com/blog/refresh-tokens-what-are-they-and-when-to-use-them/`